iree/compiler/Dialect/IREE/IR/IREEOps.td

```diff
@@ -75,6 +75,38 @@ def IREE_PlaceholderOp : IREE_Op<"placeholder", [MemoryEffects<[MemAlloc]>]> {
   let assemblyFormat = [{ `for` $purpose attr-dict `:` type($output) }];
 }
 
+def IREE_ThreadIdOp : IREE_Op<"thread_id"> {
```

Comment:
Rename to WorkgroupID to match vulkan/spirv/iree HAL terminology (or WorkgroupXYZ, as ID is sometimes overloaded to mean the linearlized ID/index)

Reply:
what about `workgroup_coord` to stay the same if we want beyond xyz?

Reply:
We likely won't (given that all GPU/CUDA/etc use XYZ) - but workgroup_id would work